packages/atomic/.storybook/default-story-shared.tsx

```diff
@@ -13,6 +12,10 @@ export interface DefaultStoryAdvancedConfig {
   additionalMarkup?: () => TemplateResult;
 }
 
+function camelToKebab(value: string) {
```

Comment:
Why not use the function from `utils`?

Reply:
Because it logs annoying warning about some missing types (NODE_TYPES, which is an import from that module used by another function).

And I'm too lazy to figure out the arcane config needed to make it resolve properly.